sycl/test-e2e/Basic/get_info_aspect.cpp

```diff
@@ -1,6 +1,7 @@
 /* This test checks that get_info checks its aspect and passes without ZES_ENABLE_SYSMAN=1.*/
 // REQUIRES: gpu-intel-dg2, level_zero
-// RUN: %{build} -o %t.out
+// RUN: %if linux %{ %{build} -DEXPECT_PASSED -o %t.out %}
```

Comment:
I think it would be better to just mark this test with: 
// XFAIL: windows 
// XFAIL-TRACKER: tracker number for the issue so that we don't forget to reenable to the test on Windows when the issue is fixed.

Reply:
Updated in recent patch.